frontend/src/component/releases/ReleasePlanTemplate/ReleasePlanTemplateAddStrategyForm.tsx

```diff
@@ -127,9 +191,35 @@ export const ReleasePlanTemplateAddStrategyForm = ({
                     )}
                 </StyledTitle>
             </StyledHeaderBox>
+            {!BuiltInStrategies.includes(strategy.name || 'default') && (
+                <StyledAlertBox>
+                    <Alert severity='warning'>
+                        Custom strategies are deprecated. We recommend not
```

Comment:
Given that custom strategies are deprecated, should we support them here at all?

Reply:
IMO we can do the same as we do in feature strategies for now and return to this later. But it's good to bring it up.